src/2sv/codes/Intro.vue

```diff
@@ -39,8 +39,8 @@ export default {
     mfa,
   }),
   computed: {
-    hasOtherTypes: vm => vm.mfa.totp.id || vm.mfa.u2f.id,
-    hasNoMfa: vm => ! (vm.mfa.totp.id || vm.mfa.u2f.id || vm.mfa.backup.id),
+    hasOtherTypes: vm => vm.mfa.totp.id || vm.mfa.u2f.id || vm.mfa.webauthn.id,
```

Comment:
maybe there should be an open issue to come back and remove the `u2f` at some point.

Reply:
Yeah, it might even be included with this PR, there's still some discussion on what to do with u2f -> webauthn in the database